### DISCUSSION TITLE
Blog posts style on the homepage

### DISCUSSION THREAD

**Discussion by kszpieg** [uPagge/uBlogger#83] [Q&A] [answered]
Hello @uPagge 

I have a question about the style of posts on the homepage. I know that this theme is based on LoveIt. I checked this theme too and I like how posts are showed on the homepage. So I'm wondering wherein is a file with code where it's possible to change of design these things?

**Comment by uPagge** [comment, marked as answer]
![изображение](https://user-images.githubusercontent.com/40397740/115539288-74ee2600-a2a5-11eb-8d8c-636cc5480f57.png)

summary.html

**Comment by Sp-cy** [comment]
@kszpieg  

How did this go? I'm planning on doing the same thing. I also love this theme overall but I've been glaring at loveit's homepage for too long.

**Reply by Sp-cy** [reply]
I thought this was gonna be way easier than I experienced. I Failed.
